src/OpenApi/test/Microsoft.AspNetCore.OpenApi.Tests/Extensions/OpenApiEndpointRouteBuilderExtensionsTests.cs

```diff
@@ -29,11 +29,13 @@ public void MapOpenApi_ReturnsEndpointConventionBuilder()
         Assert.IsAssignableFrom<IEndpointConventionBuilder>(returnedBuilder);
     }
 
-    [Fact]
-    public void MapOpenApi_SupportsCustomizingPath()
+    [Theory]
```

Comment:
Do any of these tests actually validate it is actually YAML? They check the content type and that a OpenAPI parser can parse the content, but if you replaced the writer implementation in the YAML block with the JSON one would they still pass?

Reply:
Yep, it would still pass if the writer used was a JSON one. Since JSON is valid YAML, if we wanted to verify that the emitted content was actually YAML we'd have to perhaps do some string-based check to see if it contained the correct starting characters compared to a JSON output?

Reply:
TIL JSON is YML, but that still feels wrong to me that the tests wouldn't detect the content not being the requested format.